package.json

```diff
@@ -7,16 +7,16 @@
     "@rails/actioncable": "^6.0.3-5",
     "@rails/activestorage": "^6.0.3-5",
     "@rails/ujs": "^6.0.3-5",
-    "@rails/webpacker": "^5.1.1",
+    "@rails/webpacker": "^5.4.2",
```

Comment:
I think the webpacker gem version should match this one, can you update it on the `Gemfile`?

Reply:
Done!